docs/sources/reference/run.md

```diff
@@ -1122,8 +1122,10 @@ container's `/etc/hosts` entry will be automatically updated.
 
 ## VOLUME (shared filesystems)
 
-    -v=[]: Create a bind mount with: [host-dir]:[container-dir]:[rw|ro].
-           If "container-dir" is missing, then docker creates a new volume.
+    -v=[]: Create a bind mount with: [host-dir:]container-dir[:rw|ro].
```

Comment:
Given that this approach (ie, a code-block describing the flag/option) is already used extensively in this document, I think this is fine (for consistency).